crates/biome_js_analyze/src/lint/nursery/use_sorted_classes/sort.rs

```diff
@@ -56,6 +98,16 @@ impl ClassInfo {
 // This comparison function follows a very similar logic to the one in Tailwind CSS, with some
 // simplifications and necessary differences.
 fn compare_classes(a: &ClassInfo, b: &ClassInfo) -> Ordering {
+    // Classes with arbitrary variants go last
+    if let Some(has_arbitrary_variants) = a.cmp_has_arbitrary_variants(b) {
```

Comment:
nit: a method called `*has` should return a `bool`, logically speaking. Maybe we should give a better name